### DISCUSSION TITLE
Stromversorgung

### DISCUSSION THREAD

**Discussion by hassdevil** [frickelzeugs/FingerprintDoorbell#77] [Q&A]
Hallo, danke für das coole Projekt.
Habe es bei Andreas auf seinem Kanal gesehen.

Wozu ich nichts gefunden habe, ist die Stromversorgung.
Muss ich dazu Strom zu der Klingel extra führen?

Viele Grüße
Rotator

**Comment by mickler2021** [comment]
Hi, Du kannst den Strom von der Klingel verwenden. Hier muss jedoch auf die unterschiedlichen Spannungen geachtet werden. Der Wemo mag nur 3,3V und der Klingeltrafo hat meist mehr Spannung. Gleichstrom und Wechselstrom muss auch beachtet werden. Du kannst z.B. solch Step Down Konverter verwenden um die Spannung für den Wemo runter zu regeln.

**Reply by hassdevil** [reply]
Danke für die schnelle Antwort,
ich werde mal messen was dort für eine Spannung anliegt.
Ich melde mich dann wieder, welches Bauteil ich benötige.
Vielen Dank schonmal.

**Reply by hassdevil** [reply]
![stromklingel](https://github.com/user-attachments/assets/63c4c7c1-93a5-4673-9834-d39856468bd9)
Hallo ich habe diesen Traffo verbaut, was brauch ich für einen Spannungswandler?
Der Sensor und der Esp sind bereits konfiguriert.
Ich würde dann gerne von der Klingel den Strom für den ESP nehmen, also alles hinter der Klingel verbauen.
